From: =?utf-8?b?SmFyb23DrXIgTWlrZcWh?= <mira.mikes@seznam.cz>
Date: Tue, 21 Jul 2020 17:41:45 +0200
Subject: Add keywords to desktop file

---
 src/audacity.desktop.in | 4 +++-
 1 file changed, 3 insertions(+), 1 deletion(-)

Index: audacity/src/audacity.desktop.in
===================================================================
--- audacity.orig/src/audacity.desktop.in
+++ audacity/src/audacity.desktop.in
@@ -41,6 +41,8 @@ Comment[uk]=Запис і редагув
 Comment[zh_CN]=录音和编辑音频文件
 Comment[zh_TW]=錄音和編輯音訊檔案
 
+Keywords=audio;sound;alsa;jack;editor;
+
 Icon=@AUDACITY_NAME@
 
 Type=Application
