From: Sebastian Ramacher <sramacher@debian.org>
Date: Tue, 21 Jul 2020 21:56:20 +0200
Subject: Install plugins in /usr/lib

---
 CMakeLists.txt      | 2 +-
 src/AudacityApp.cpp | 6 ++++++
 2 files changed, 7 insertions(+), 1 deletion(-)

diff --git a/CMakeLists.txt b/CMakeLists.txt
index 4ff5430..40c3618 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -380,7 +380,7 @@ set( _PREFIX "${CMAKE_INSTALL_PREFIX}" )
 set( _LIBDIR "${CMAKE_INSTALL_LIBDIR}/audacity" )
 set( _RPATH "\$ORIGIN/../${_LIBDIR}" )
 set( _DATADIR "${CMAKE_INSTALL_DATADIR}" )
-set( _PKGDATA "${_DATADIR}/audacity/" )
+set( _PKGDATA "${_LIBDIR}" )
 set( _MANDIR "${CMAKE_INSTALL_MANDIR}" )
 
 # Precreate the lib and lib64 directories so we can make then the same
diff --git a/src/AudacityApp.cpp b/src/AudacityApp.cpp
index 661ab7e..3c29ac7 100644
--- a/src/AudacityApp.cpp
+++ b/src/AudacityApp.cpp
@@ -1183,6 +1183,9 @@ bool AudacityApp::OnInit()
    FileNames::AddUniquePathToPathList(wxString::Format(wxT("%s/.%s-files"),
       home, wxT(AUDACITY_NAME)),
       audacityPathList);
+   FileNames::AddUniquePathToPathList(wxString::Format(wxT("%s/lib/%s"),
+      wxT(INSTALL_PREFIX), wxT(AUDACITY_NAME)),
+      audacityPathList);
    FileNames::AddUniquePathToPathList(wxString::Format(wxT("%s/share/%s"),
       wxT(INSTALL_PREFIX), wxT(AUDACITY_NAME)),
       audacityPathList);
@@ -1193,6 +1196,9 @@ bool AudacityApp::OnInit()
    FileNames::AddUniquePathToPathList(wxString::Format(wxT("%s/.audacity-files"),
       home),
       audacityPathList)
+   FileNames::AddUniquePathToPathList(wxString::Format(wxT("%s/lib/audacity"),
+      wxT(INSTALL_PREFIX)),
+      audacityPathList);
    FileNames::AddUniquePathToPathList(wxString::Format(wxT("%s/share/audacity"),
       wxT(INSTALL_PREFIX)),
       audacityPathList);
